typings/index.d.ts

```diff
@@ -3489,6 +3488,13 @@ declare module 'discord.js' {
     global: boolean;
   }
 
+  // tslint:disable-next-line:no-empty-interface - Merge RateLimitData interface to not have to type it again
+  interface RateLimitError extends RateLimitData {}
+  export class RateLimitError extends Error {
```

Comment:
this is a class so it has to go up, i think theres a better way to do this too, `extends Constructable<Error & RateLimitData>` afaik

Reply:
I wasn't sure where to put this, so I just put it below the existing RateLimitData. what would be a more appropriate place for it?

Reply:
Moved it below HttpError now. Constructable indeed works but then it'd be pretty awkward syntax
```ts
export type RateLimitError = Constructable<Error & RateLimitData & { 
  constructor(data: RateLimitData);
  name: 'RateLimitError';
}>
    ```